library-best-practices/JLBP-14.md

```diff
@@ -0,0 +1,16 @@
+[JLBP-14] Do not use version ranges
+-----------------------------------
+
+- If the pom.xml of your library specifies a version range instead of a
+  single version for any particular dependency, builds at different points
+  in time can get different versions of that dependency (as your dependency
```

Comment:
perhaps, can get --> see

Reply:
I think the "can" part is still necessary; if a dependency never actually updates, you won't see different versions. I will s/get/see/ though.